### PR TITLE
Import initial governance docs

### PR DESCRIPTION
**Pull request by AevaOnline**
Import and the initial governance docs, based on the google doc content approved by the TAC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+## Observation on the need for common standards
+
+Attestation features found in current offerings in the Confidential Computing
+space meaningfully improve security of confidential applications by offering
+computing parties a way to verify specific properties of each other’s
+workloads, lessening the need for blind trust that must be placed on the other
+party. However, these solutions do not yet offer a standard way for customers
+to declare, configure and authenticate attestation claims of a peer workload
+that might be running in a peer TEE (e.g., Intel SGX, AMD SEV, ARM TrustZone,
+etc. or other protected environment), they may offer partial or limited
```
Comment (thomas-fossati) [inline README.md:17]: * _"partial or limited"_ one of the two looks redundant
* adding an _"and"_ after the comma

```suggestion
etc. or other protected environment), and they may offer limited
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,54 @@
...
+
+## Observation on the need for common standards
+
+Attestation features found in current offerings in the Confidential Computing
+space meaningfully improve security of confidential applications by offering
+computing parties a way to verify specific properties of each other’s
+workloads, lessening the need for blind trust that must be placed on the other
+party. However, these solutions do not yet offer a standard way for customers
+to declare, configure and authenticate attestation claims of a peer workload
+that might be running in a peer TEE (e.g., Intel SGX, AMD SEV, ARM TrustZone,
```
Comment (thomas-fossati) [inline README.md:16]:
```suggestion
that might be running in a peer TEE (e.g., Intel SGX, AMD SEV, Arm TrustZone,
```

**Review by thomas-fossati — COMMENTED** [review]
lgtm, thanks! I've left a couple of editorial comments.

**Comment by AevaOnline** [comment]
@thomas-fossati it looks like I accidentally broke this PR while renaming the "master" -> "main" branch in the repo. Will try to restore, but may need to re-open. I'll include your feedback, thanks!

**Comment by shnwc** [comment]
LGTM.

**Review by thomas-fossati — APPROVED** [review]

**Comment by AevaOnline** [comment]
following discussion on the call, with approval from Mike thus making 3/5ths, merging this.

**Pull request merged**